rock/python.rb

```diff
@@ -0,0 +1,50 @@
+require 'open3'
+
+module Rock
+    # Identify the available python version, per default checks
+    # Autobuild.programs['python'] setting
+    # @param[String] python_bin Path to a custom python version
+    def self.find_python_bin(python_bin = nil)
+        if !python_bin
+            python_bin = Autobuild.programs['python']
```

Comment:
You could store it in the configuration file, to be consistent with the way the Ruby stuff is done.

Also, I'd suggest to auto-cache the detected value there to avoid calling which repeatedly.